src/js/common/schemaform/widgets/TextWidget.jsx

```diff
@@ -12,7 +12,7 @@ export default function TextWidget(props) {
         maxLength={props.schema.maxLength}
         autoComplete={props.options.autocomplete || false}
         className={props.options.widgetClassNames}
-        value={props.value || ''}
```

Comment:
Could change this to `value={props.value || undefined}` but it seemed a little redundant.